pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -229,18 +229,9 @@ stdenv.mkDerivation rec {
       The Programs handle Schematic Capture, and PCB Layout with Gerber output.
     '';
     license = lib.licenses.gpl3Plus;
-    maintainers = with lib.maintainers; [ evils kiwi ];
-    # kicad is cross platform
+    maintainers = with lib.maintainers; [ evils ];
     platforms = lib.platforms.all;
     broken = stdenv.isDarwin;
-
-    hydraPlatforms = if (with3d) then [ ] else platforms;
-    # We can't download the 3d models on Hydra,
```

Comment:
this comment makes it sound like the downloading itself is the issue
TBD if this actually works on hydra now...

Reply:
derivation output size (total of all outputs) is limited to 3G, this change should work